sympy/ntheory/factor_.py

```diff
@@ -19,6 +19,16 @@
 from .primetest import isprime
 from .generate import sieve, primerange, nextprime
 
+"""
+# Note: This list should be updated whenever new Mersenne primes are found.
+# Refer: https://www.mersenne.org/
+"""
```

Comment:
No need to add triple quotes. Remove it and also the blank line below this.
Something like:

```
# Note: .....
# Refer: .....
MERSENNE_PRIME_EXPONENTS = (.....
```